packages/pwa-kit-dev/src/configs/webpack/plugins.js

```diff
@@ -17,13 +17,18 @@ import fs from 'fs'
  * @returns {webpack.NormalModuleReplacementPlugin}
  */
 export const createModuleReplacementPlugin = (projectDir) => {
+    // Helper function to create a RegExp object from a string
     const makeRegExp = (str, sep = path.sep) => {
         // Replace unix paths with windows if needed and build a RegExp
         if (sep === '\\') {
             str = str.replace(/\//g, '\\\\')
         }
         return new RegExp(str)
     }
+
+    // List of overridable paths
+    // path: The RegExp that matches the path to the overridable component
+    // newPath: The path to the component in the project directory
```

Comment:
Since it's not always a component...

```suggestion
    // path: The RegExp that matches the path to the overridable file
    // newPath: The corresponding path in the project directory
```